openmmtools/forces.py

```diff
@@ -21,6 +21,7 @@
 import logging
 import math
 import re
+import yaml
```

Comment:
Are we using this module here?

Reply:
Oh, no. I had that in as part of other serialization attempts. I'll fix it.